documentation/book/proc-configuring-maintenance-time-windows.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// assembly-maintenance-time-windows.adoc
+
+[id='proc-configuring-maintenance-time-windows-{context}']
+= Configuring maintenance time windows
+
+.Prerequisites
+
+* An {ProductPlatformName} cluster
+* A running Cluster Operator
```

Comment:
A running Cluster Operator > The Cluster Operator is running.

Add full stops after each prereq.